src/Symfony/Component/Serializer/Normalizer/CacheableSupport.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+/**
+ * The return value of supports*() methods  in {@see NormalizerInterface} will be cached by type and format, based on the returned SUPPORT_* value.
```

Comment:
```suggestion
 * The return value of supports*() methods in {@see NormalizerInterface} and {@see DenormalizerInterface}. 
 * Tells if the result should be cached based on type and format.
```
WDYT?

Reply:
Looks good to me (I now see the 'commit suggestion' button, should've used that..)